airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -467,20 +454,15 @@ class DataprocCreateClusterOperator(BaseOperator):
     :type impersonation_chain: Union[str, Sequence[str]]
     """
 
-    template_fields = (
```

Comment:
Is there a particular reason to remove `template_fields` for this operator entirely?

Reply:
No particular reason, I will move it to align it with other operators